app/models/miq_expression.rb

```diff
@@ -905,6 +894,10 @@ def field_from_virtual_reflection?(field)
     col_details[field][:virtual_reflection]
   end
 
+  def attribute_supported_by_sql?(field)
+    col_details[field][:sql_support]
```

Comment:
Hmm. I can only assume either a) bug, or b) unnecessary, since I don't *believe* that PR added any new callers of this method. The evil part of me wants to go with (b), since there's no test to prove that it's needed :trollface: